tasks/main.yml

```diff
@@ -1 +1,17 @@
 # role tasks
+
+- name: Run shred to write zeros to the disk
+  become: "{{ disk_wipe_become }}"
+  become_user: "{{ disk_wipe_become_user }}"
+  command: >
+    shred -v -n 1 --random-source=/dev/zero -s
+    "{{ disk_wipe_write_size }}"
+    "{{ item }}"
+  with_items: "{{ disk_wipe_disks }}"
+
+- name: Run wipefs to clear the disk signatures
+  become: "{{ disk_wipe_become }}"
+  become_user: "{{ disk_wipe_become_user }}"
+  ignore_errors:
```

Comment:
Is this syntactically correct?  Do we need to ignore errors?

Reply:
Yeah my bad missed some clean up.